Runtimes/NET/BepInEx.NET.Framework.Launcher/BepInEx.NET.Framework.Launcher.csproj

```diff
@@ -16,6 +16,7 @@
     <ItemGroup>
         <PackageReference Include="HarmonyX" Version="2.10.1"/>
         <PackageReference Include="Microsoft.NETFramework.ReferenceAssemblies" Version="1.0.3" PrivateAssets="All"/>
+        <PackageReference Include="MonoMod.RuntimeDetour" Version="22.5.1.1" />
```

Comment:
Why? It should be picked up from `BepInEx.Preloader.Core`

Reply:
Accidentally forgot to remove this when segmenting changes